weareinreach/inreach-catalog
# Support grouped runs

## Description

In this PR I break down the Automation tests into groups for Circle CI. Local Testing will remain the same, but in Circle CI it will break it up into groups and run them parallel. Since each instance will have its own API and Docker DB. Tests will not crash with each other.

## Asana ticket: Second part of: https://app.asana.com/0/1132189118126148/1201784120973086

## PR Checklist




## How to Test

Check Circle CI to see e2e-1 all the way to e2e-4. NOTE: Tests are still failing and will be fixed on next PR.

DEPENDENT ON PR: https://github.com/asylum-connect/asylumconnect-catalog/pull/208
